patterns-base/src/main/resources/docs/description/Custom_Scala_EmptyForBlock.md

```diff
@@ -0,0 +1,12 @@
+Avoid using empty blocks on for statements
+
+     for( ... )
+     {} // <-- avoid this
+
+Or
+	 for{ ... }
+     yield {} // <-- avoid this
```

Comment:
Actually, taking a better look at this I think that this pattern should be called "EmptyYieldBlock" instead and we could also have this pattern but just for empty blocks in for statemts, as you showed in the first example. What do you think, @DReigada @andreaTP @rtfpessoa ?

Reply:
I thought about it, when I was creating the pattern, but I just searched on Scala documentation, and found [For and For-Yield Loops][1] merged all together.

[1]: https://docs.scala-lang.org/overviews/quasiquotes/expression-details.html#for-and-for-yield-loops